src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.cs

```diff
@@ -139,7 +139,7 @@ public partial class FixAllContext
         /// <summary>
         /// Gets all the diagnostics in the given document filtered by <see cref="DiagnosticIds"/>.
         /// </summary>
-        public async Task<ImmutableArray<Diagnostic>> GetDocumentDiagnosticsAsync(Document document)
+        public Task<ImmutableArray<Diagnostic>> GetDocumentDiagnosticsAsync(Document document)
```

Comment:
I would not change this one.

Reply:
OK, I'll revert, but is this technically invalid or just bad for code readability?